mmdet3d/datasets/transforms/loading.py

```diff
@@ -387,7 +387,7 @@ def _remove_close(self,
         if isinstance(points, np.ndarray):
             points_numpy = points
         elif isinstance(points, BasePoints):
-            points_numpy = points.tensor.numpy()
```

Comment:
An instance of BasePoints does not have `numpy()` method ? Please check the all.

Reply:
The same above.